src/docs/product/data-management-settings/scrubbing/server-side-scrubbing.mdx

```diff
@@ -27,7 +27,9 @@ With it enabled, Sentry will scrub the following:
   - github_token
   - privatekey
   - private_key
-- Any values whose keynames are contained in the list of additional fields in your Project Settings.
+- Values that contain strings in, or whose keynames are listed in, Sensitive Fields in your Project Settings.
```

Comment:
Do you mean the "Additional Sensitive Fields" field in [Project] > Settings > Security & Privacy, or something else?
```suggestion
- Values that contain strings in, or whose keynames are listed in, [Project] > Settings > Security & Privacy in "Additional Sensitive Fields".
```

Reply:
yeah that 👍